package.json

```diff
@@ -41,8 +41,10 @@
     "build": "REACT_APP_MSW_ON=0 react-scripts build",
     "test": "react-scripts test",
     "cy:run": "cypress run --e2e",
+    "cy:run:spec": "cypress run --e2e --config video=false --spec $RUNNER_SPEC",
     "cy:dev": "cypress open --e2e",
     "test:e2e": "CYPRESS_USE_FAKE=1 start-server-and-test 'BROWSER=none npm run start:fake' '3000|3002' cy:run",
+    "test:e2e:spec": "CYPRESS_USE_FAKE=1 start-server-and-test 'BROWSER=none npm run start:fake' '3000|3002' cy:run:spec",
```

Comment:
please also update the github workflow that uses this (ie runs tests in parallel)

Reply:
yes will do (was planning to do in a separate PR to keep commit history for the repo locale actions sane)